### PR TITLE
Create array_search() doc #3047

### PR DESCRIPTION
**Pull request by reganmarie**
### Description

- Closes issue #3047 
- Created array_search.md on file path: `docs/content/php/concepts/arrays/terms/array-search/array-search.md`
- Added Title, Description, Subjects, Tags, and Catalog Content
- Added Description, Syntax, Example, and Codebyte example
- Ran through 3 online plagiarism checkers to ensure 100% unique wording and description, passed with 100%
- Followed content and style guidelines

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/array-search/array-search.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+The **`array_seach()`** function searches an array for a given value and returns the first matching key for that value.
+
+## Syntax
+
+```pseudo
+array_search(value, array, strict)
+```
+
+- `value`: The value to search for.
+> **Note:** String values are case-sensitive for this function.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-search/array-search.md:25]: I would suggest moving the note after all the parameters

File: content/php/concepts/arrays/terms/array-search/array-search.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Tags:
+  - 'Arrays'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_seach()`** function searches an array for a given value and returns the first matching key for that value.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-search/array-search.md:16]:
```suggestion
The **`array_search()`** function searches an array for a given value and returns the first matching key for that value.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@reganmarie Just added a couple of comments. Please update, and tag me when you've finished making the changes. Thanks

File: content/php/concepts/arrays/terms/array-search/array-search.md (hunk 1 above)
Comment (reganmarie) [reply content/php/concepts/arrays/terms/array-search/array-search.md]: Fixed!

**Comment by reganmarie** [comment]
Fixed the spelling of search in line 16 and moved the note under all of the parameters! @caupolicandiaz - I realized that you may want the link directly for the newest commit :) https://github.com/Codecademy/docs/pull/3161/commits/98058b39ffd6cc52da523fd0505c43d69c35bc7d

**Review by caupolicandiaz — APPROVED** [review]
@reganmarie Looks good, I just made a couple of minor formatting edits directly. This is ready for a second review.

**Comment by reganmarie** [comment]
Wonderful, thank you @caupolicandiaz ! Please let me know if there's anything else I can do in the meantime :)

**Review by SSwiniarski — APPROVED** [review]
@reganmarie Looks good.

**Comment by reganmarie** [comment]
Thank you @SSwiniarski ! :)

**Pull request merged**